maas-schemas/schemas/maas-backend/tsp-auth/verify/response.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/tsp-auth/verify/response.json",
+  "description": "Response schema for tsp-auth verify",
+  "type": "object",
+  "properties": {
+    "allProductsVerified": {
+      "type": "boolean",
+      "description": "A flag indicating whether all products has been verified"
+    },
+    "products": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "$ref": "http://maasglobal.com/core/booking.json#/definitions/tspId"
+          },
+          "verified": {
+            "type": "boolean"
+          }
+        }
+      }
+    },
+    "verificationFailureKey": {
+      "description": "Failure key enum, identifying reason for the verification failure",
+      "type": "string",
+      "enum": ["UNKNOWN_ERROR", "NO_TOKEN_FOR_THE_USER", "INVALID_USER_TOKEN"]
```

Comment:
This could be moved to definitions and then you can reference it here.

Reply:
Fixed.